dev/snippets/nav/bar_basic.cljs

```diff
@@ -1,9 +1,11 @@
 #_
 (:require [om-bootstrap.button :as b]
-          [om-bootstrap.nav :as n])
+          [om-bootstrap.nav :as n]
+          [om-tools.dom :as d :include-macros true])
 
 (n/navbar
- {}
+ {:brand (d/a {:href "#"}
```

Comment:
So now you can either just add a :brand or :toggle-button to your Navbar props, and specify :collapsible? true in your Nav props, or specify a :toggle-nav-key in your Navbar props and a corresponding :key in your Nav props